### PR TITLE
AST-942 - Global variables with astra prefix NamingConvention

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- PR  is about resolving PHPCS NamingConvention ruleset errors for variables in Astra.
- Main PR - https://github.com/brainstormforce/astra/pull/3247

### Types of changes
- Code refactor 

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: template-parts/header/builder/header-row.php [hunk 1]
```diff
@@ -6,84 +6,83 @@
  */
 
 if ( astra_wp_version_compare( '5.4.99', '>=' ) ) {
-	$row = wp_parse_args( $args, array( 'row' => '' ) );
-	$row = $row['row'];
+	$astra_header_row = wp_parse_args( $args, array( 'row' => '' ) );
+	$astra_header_row = $astra_header_row['row'];
 } else {
```
Comment (sagarb90) [inline template-parts/header/builder/header-row.php:11]: Improvements
```
$astra_header_row = get_query_var( 'row' );
if ( astra_wp_version_compare( '5.4.99', '>=' ) ) {
	$astra_header_row = wp_parse_args( $args, array( 'row' => '' ) );
	$astra_header_row = $astra_header_row['row'];
}

Comment (imnavanath) [reply template-parts/header/builder/header-row.php]: Done @sagarb90 you can review it from here - https://github.com/brainstormforce/astra/pull/3316/commits/0041dbb155835937e8351206c0ac00fc6dc2d4a1
Also I've done the same for all possible instances.

**Review by sagarb90 — APPROVED** [review]

**Review by abhishek-bsf — APPROVED** [review]

File: template-parts/footer/builder/components.php [hunk 2]
```diff
@@ -149,17 +146,17 @@
 		break;
 
 	case 'divider-1':
-		$layout_class = astra_get_option( 'footer-divider-1-layout' );
+		$astra_fb_layout_class = astra_get_option( 'footer-divider-1-layout' );
```
Comment (premanshup) [inline template-parts/footer/builder/components.php:149]:
```suggestion
		$astra_fb_divider_layout_class = astra_get_option( 'footer-divider-1-layout' );
```

**Review by premanshup — APPROVED** [review]

File: template-parts/footer/builder/components.php (hunk 2 above)
Comment (imnavanath) [reply template-parts/footer/builder/components.php]: Done 👍

**Pull request merged**